libcxx/include/__format/format_arg_store.h

```diff
@@ -228,15 +228,15 @@ _LIBCPP_HIDE_FROM_ABI void __store_basic_format_arg(basic_format_arg<_Context>*
   ([&] { *__data++ = __format::__create_format_arg<_Context>(__args); }(), ...);
 }
 
-template <class _Context, size_t N>
```

Comment:
As mentioned in #76829 we typically name these `_Np`. I don't see a reason to deviate in this case.